core/block/export/writer.go

```diff
@@ -105,10 +113,14 @@ func (d *zipWriter) Path() string {
 	return d.path
 }
 
-func (d *zipWriter) WriteFile(filename string, r io.Reader) (err error) {
+func (d *zipWriter) WriteFile(filename string, r io.Reader, lastModifiedDate int64) (err error) {
 	d.m.Lock()
 	defer d.m.Unlock()
-	zf, err := d.zw.Create(filename)
+	zf, err := d.zw.CreateHeader(&zip.FileHeader{
```

Comment:
need to fallback to time.Now() in case of 0

Reply:
Fixed